### PR TITLE
support-stripe component

### PR DESCRIPTION
**Pull request by noslouch**
[ticket](https://jira.wnyc.org/browse/RT-623)

### CODE REVIEW

File: app/styles/nypr-ui/_support-stripe.scss [hunk 1]
```diff
@@ -0,0 +1,106 @@
+.support-stripe {
+  display: flex;
+  position: relative;
+
+  font-family: "Open Sans", sans-serif;
+}
+
```
Comment (jkeen) [inline app/styles/nypr-ui/_support-stripe.scss]: I thought we were doing `.nypr-[name]` for components in this repo?

Comment (jkeen) [reply app/styles/nypr-ui/_support-stripe.scss]: Oh, nevermind, we do it sometimes, not all the times 🤷‍♂️

**Comment by jkeen** [comment]
Rebased with master and fixed a test

File: addon/templates/components/support-stripe.hbs [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{{#if link}}
```
Comment (noslouch) [inline addon/templates/components/support-stripe.hbs]: this confuses the API for this component. It yields out a link component and also accepts a link parameter? What if they get different values? Which link should take precedence? I think there should only be 1 input for an href here. Also I think nesting `<a/>` tags is generally considered bad practice.

Would you mind refactoring this a bit? I think it's safe to assume that this should always link out to an external resource.

- the root element (i.e. `tagName` can be `a`)
- change the name of the yielded `link` component to something like `label` and make it only accept a text argument, i.e. the `href` value should be a top-level argument
- you shouldn't need this conditional in the template now

Comment (mike-hearn) [reply addon/templates/components/support-stripe.hbs]: Fixed in:

* https://github.com/nypublicradio/nypr-ui/pull/78/commits/8a482c540263662a850374be29e8bb98622be4d6
* https://github.com/nypublicradio/wnyc-web-client/pull/213/commits/aac4fad7a5ff0502225f3a22038a3c66f575eee6

**Pull request merged**